app/src/main/java/com/example/puzzlepic/dto/Picture.kt

```diff
@@ -5,13 +5,19 @@ import com.google.gson.annotations.SerializedName
 import kotlinx.android.parcel.Parcelize
 
 
-data class Picture(@SerializedName("id") var pictureID : String = "", @SerializedName("alt_description") var pictureTitle : String, @SerializedName("created_at") var date : String, var location : String, var notes : String,
-                        var puzzleID : Int = 0, @SerializedName("urls") var urls : UnsplashPhotoUrls ){
-        override fun toString() : String {
-            return pictureTitle
-            return date
-        }
+data class Picture(
+    @SerializedName("id") var pictureID : String = "",
+    @SerializedName("alt_description") var pictureTitle : String,
+    @SerializedName("created_at") var date : String,
+    var location : String, var notes : String,
+    var puzzleID : Int = 0,
+    @SerializedName("urls") var urls : UnsplashPhotoUrls
+) {
```

Comment:
I divided each primary constructor parameter into its own separate line because according to the kotlin coding convention's class-headers rule, "Classes with longer headers should be formatted so that each primary constructor parameter is in a separate line with indentation." I separated them and added indentation before each parameter. Now it visually looks better and organized! Check out the rule here - https://kotlinlang.org/docs/coding-conventions.html#class-headers